evalutils/templates/processor/cookiecutter.json

```diff
@@ -0,0 +1,17 @@
+{
+    "processor_name": "",
+    "diag_ticket": "",
+    "package_name": "{{ cookiecutter.processor_name|replace(' ', '') }}",
+    "evalutils_name": "",
+    "evalutils_version": "",
+    "docker_base_container": "python:3.7-slim",
+    "dev_build": 0,
+    "requirements":  {
+        "cpu_count": 1,
+        "cpu_capabilities": [],
+        "memory": "1G",
```

Comment:
Should be an `int`

Reply:
What exactly should be an `int` the `memory` field?

Reply:
The memory field string format is now validated on cli.py. I would like to keep this format, since it is also directly supported by docker.